templates/elfinder/file_manager.html

```diff
@@ -199,6 +199,9 @@
                 }else if ( kokoLang.startsWith("zh")) {
                     lng = 'zh_CN'
                 }
+                else if (kokoLang.startsWith('pt')) {
+                    lng = 'pt_BR';
+                }
                 if (lng !== 'en') {
         		    $.ajax({
         		    	url : '/koko/static/plugins/elfinder/i18n'+'/elfinder.'+lng+'.js',
```

Comment:
The provided code contains some minor changes that aim to fix inconsistencies between locales. It is recommended to ensure these settings have been properly tested across different regions and cultures before moving forward, such as adjusting for potential localization issues.